digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/dao/RoadAddressDAO.scala

```diff
@@ -298,6 +298,12 @@ object RoadAddressDAO {
     else
       ""
 
+    val valid = if(includeValid) {
+      "AND (ra.end_date is null OR ra.end_data >= sysdate) AND ra.terminated = 0 "
```

Comment:
Should not include a date check: all road address history should be moved with vvh change table to the new link. Instead only termination should be checked.

Reply:
use includeTerminated flag and change to 
if (!includeTerminated) {
  "AND ra.terminated = 0 "
...